asserts/store.go

```diff
@@ -0,0 +1,116 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2014-2015 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+// Package asserts implements snappy assertions and a store
+// abstraction for managing and holding them.
+package asserts
+
+import (
+	"bytes"
+	"encoding/hex"
+	"errors"
+	"os"
+	"path/filepath"
+
+	"golang.org/x/crypto/openpgp/packet"
+
+	"github.com/ubuntu-core/snappy/helpers"
+)
+
+// StoreConfig for an assertion store.
+type StoreConfig struct {
+	// store backstore path
+	Path string
+}
+
+// AssertStore holds assertions and can be used to sign or check
+// further assertions.
+type AssertStore struct {
```

Comment:
store doesn't have remote implication for me but I understand it's used already for something that does in snappy.  Database is ok with me, it's also the term Prolog uses for its fact repository.